101-analysis-services-create/azuredeploy.json

```diff
@@ -0,0 +1,56 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "serverName": {
+      "type": "string",
+      "metadata": {
+        "description": "The name of the Azure Analysis Services server to create. Server name must begin with a letter, be lowercase alphanumeric, and between 3 and 63 characters in length."
+      }
+    },
+    "skuName": {
+      "type": "string",
+      "defaultValue": "S0",
+      "metadata": {
+        "description": "The sku name of the Azure Analysis Services server to create. Choose from: B1, B2, D1, S0, S1, S2, S3, S4, S8, S9. Some skus are region specific. See https://docs.microsoft.com/en-us/azure/analysis-services/analysis-services-overview#availability-by-region"
+      }
+    },
+    "capacity": {
+      "type": "int",
+      "defaultValue": 1,
+      "metadata": {
+        "description": "The total number of query replica scale-out instances. Scale-out of more than one instance is supported on selected regions only. See https://docs.microsoft.com/en-us/azure/analysis-services/analysis-services-overview#availability-by-region"
+      }
+    },
+    "firewallSettings": {
+      "type": "object",
+      "defaultValue": {},
+      "metadata": {
+        "description": "The inbound firewall rules to define on the server. If not specified, firewall is disabled."
+      }
+    },
+    "backupBlobContainerUri": {
+      "type": "string",
+      "defaultValue": "",
+      "metadata": {
+        "description": "The URI to an Azure Blob Storage container. Required only if you intend to use the backup/restore functionality. See https://docs.microsoft.com/en-us/azure/analysis-services/analysis-services-backup"
```

Comment:
SAS URI

Reply:
yes, need more info here?  Does it need to be public? can it be sas? and if so what are the perms?